### PR TITLE
Update intro graph algos for sandbox support

### PR DESCRIPTION
**Pull request by tomasonjo**

### CODE REVIEW

**Comment by Mogztter** [comment]
Hey @tomasonjo 

Here's a proposal to branch the navigation:

setup-branching

"Environment Setup" has two sub items on the left menu:
- Local Setup
- Sandbox Setup

The page "Environment Setup" has two "next links" and a previous link (to go back to the "Introduction to Graph Data Science library" page).

The "Sandbox Setup" page has a link in the "Summary" section to go the "Graph Algorithms Workflow" page:

sandbox-setup-nav-next

In my opinion, the links on the "Environment Setup" are fine.

On the other hand, on the "Sandbox Setup", the "Check Answers" button should be above the "Summary" section, otherwise it's easy to miss the "the hand-on exercises" link to go to the next page.

Thoughts?

Here's the diff if you want to take a look: https://github.com/tomasonjo/training-v3/compare/master...Mogztter:iga-navigation-branching?expand=1

**Comment by tomasonjo** [comment]
Yes, I would agree that the check button should be above the summary... Currently, the dataset and questions are duplicated on both the Local and Sandbox setup. Maybe it would make sense to create a new page for that so that we don't duplicate content and have to maintain both. In this case, we would split for example the local setup page at https://github.com/neo4j-contrib/training-v3/pull/251/files#diff-09d8f0b901987ec4a49fa33500230cecdac888e3e0ad0c89c16c4542b1d85093R118 and then create a page with datasets that both local and sandbox developer environments setup point to. 

cc @ElaineRosenberg

**Comment by ElaineRosenberg** [comment]
This is looking good Guillaume and Tomaz. I agree with Tomaz that we need to try to eliminate duplication of content.

**Comment by tomasonjo** [comment]
I have merged the PR and will do some updates to remove duplicating content. I'll ping you both to do the last check after.

**Comment by tomasonjo** [comment]
I've added another PR. There are some things missing it seems as the navigation bar is a bit buggy, but from the content perspective I think it should work

**Comment by Mogztter** [comment]
> In this case, we would split for example the local setup page at #251 (files) and then create a page with datasets that both local and sandbox developer environments setup point to.

That's a good idea :+1: 

> I've added another PR. There are some things missing it seems as the navigation bar is a bit buggy, but from the content perspective I think it should work.

What do you mean by a bit buggy? Are you referring to the fact that some items have a circle (bullet) and some items don't?

**Comment by tomasonjo** [comment]
I had to fix the nav.adoc file, now it works fine... The only thing that doesn't work for me is the link from local/sandbox setup to database-import. And we need to remove the check results from the setup page, not sure how

File: modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+You can start it by clicking on the *Open* button dropdown and selecting NEuler.
+
+image::open-neuler.png[local and global graph analytics,width=500, align=center]
+
+
+[.summary]
+== Summary
+
+You have now prepared your development enviroment for performing
+xref:06-iga-40-graph-dataset-import.adoc[the dataset import] for this course.
```
Comment (Mogztter) [inline modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc:46]:
```suggestion
xref:06-iga-40-dataset-import.adoc[the dataset import] for this course.
```

File: modules/4.0-intro-graph-algos/modules/ROOT/pages/05a-iga-local-setup.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
+= Local Setup
+:page-slug: 05a-iga-40-local-setup
+:page-layout: training
+:page-quiz:
```
Comment (Mogztter) [inline modules/4.0-intro-graph-algos/modules/ROOT/pages/05a-iga-local-setup.adoc:4]: We should remove the attribute `:page-quiz` if the page does not contain a quiz.

```suggestion
```

File: modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
+= Sandbox Setup
+:page-slug: 05a-iga-40-sandbox-setup
```
Comment (Mogztter) [inline modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc:2]:
```suggestion
:page-slug: 05b-iga-sandbox-setup
```

File: modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+:page-layout: training
+:page-quiz:
```
Comment (Mogztter) [inline modules/4.0-intro-graph-algos/modules/ROOT/pages/05b-iga-sandbox-setup.adoc:4]: ditto

```suggestion
```

**Comment by tomasonjo** [comment]
@Mogztter thanks for the help. I think it's fine now. @ElaineRosenberg if you want to do a final check

**Comment by ElaineRosenberg** [comment]
did the merge manually

**Pull request closed without merging**